tests/testthat/test-get.R

```diff
@@ -30,3 +30,21 @@ test_that("get_constant_vars Test 2: with ignore_vars", {
     vars(USUBJID, AGE)
   )
 })
+
+# get_duplicates ----
+## Test 1: x atomic vector ----
+test_that("get_duplicates Test 1: x atomic vector", {
+  x <- c("a", "a", "b", "c", "d", "d", 1, 1, 4)
+
+  expect_equal(
+    get_duplicates(x),
+    c("a", "d", 1)
+  )
+})
+
+## Test 2: x not atomic vector ----
+test_that("get_constant_vars Test 2: x not atomic vector", {
+  x <- list("a", "a", "b", "c", "d", "d", 1, 1, 4)
+
+  expect_error(get_duplicates(x))
+})
```

Comment:
I think this test is overkill as it test `assert_atomic_vector()` rather than `get_duplicates()`. SO what I'd propose instead is to add a test for `assert_atomic_vector()`

Reply:
Hi Thomas, good point! I moved the test to test-assertions. Please let me know if there is anything else I should add/edit.